themes/default/content/pulumi-up/_index.md

```diff
@@ -23,7 +23,7 @@ event_sections:
 
     - title: The Tools
       act: II
-      speakers: [ "luke-hoban", "sophia-parafina", "komal-ali" ]
+      speakers: [ "luke-hoban", "lee-zen", "komal-ali" ]
```

Comment:
I see we have this layout on the page now:

![image](https://user-images.githubusercontent.com/274700/115638698-bee80380-a2c7-11eb-8d07-9ec025bb78f1.png)

... but is it right? Should Komal's image be below Luke's? It looks somehow off -- although I'm guessing the intent may be to "center" the second row? (If so, ignore me.)

Reply:
👍 Yeah I'll need to play with the alignment to try and clean this up.